pytorch_lightning/core/lightning.py

```diff
@@ -1392,12 +1392,15 @@ def get_progress_bar_dict(self):
         """
         # call .item() only once but store elements without graphs
         running_train_loss = self.trainer.train_loop.running_loss.mean()
-        avg_training_loss = (
-            running_train_loss.cpu().item()
-            if running_train_loss is not None
-            else float("NaN")
-        )
-        tqdm_dict = {"loss": "{:.3g}".format(avg_training_loss)}
+        if self.trainer.train_loop.automatic_optimization:
+            avg_training_loss = (
+                running_train_loss.cpu().item()
+                if running_train_loss is not None
+                else float("NaN")
+            )
+            tqdm_dict = {"loss": "{:.3g}".format(avg_training_loss)}
+        else:
+            tqdm_dict = {}
```

Comment:
We can reorder this code to support manual optimization users who set `running_losses` themselves:

```suggestion
        avg_training_loss = None
        if running_train_loss is not None:
            avg_training_loss = running_train_loss.cpu().item()
        elif self.trainer.train_loop.automatic_optimization:
            avg_training_loss = float('NaN)

        tqdm_dict = {}
        if avg_training_loss is not None:
            tqdm_dict["loss"] = f"{avg_training_loss:.3g}"
```

Reply:
Hey @carmocca, I don't except people using `running_losses` as it is an internal object. But I like the idea :)